tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml

```diff
@@ -3,306 +3,307 @@
 
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 ---
-- block:
+- name: Block of tests
+  block:
   - name: Test an API key is provided
-    fail:
+    ansible.builtin.fail:
       msg: Please define an API key
     when: auth_key is not defined
 
   - name: Create network
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+      org_name: '{{ test_org_name }}'
+      net_name: '{{ test_net_name }}'
       type: appliance
       timezone: America/Chicago
     delegate_to: localhost
     register: create_net_appliance
 
   - name: Test net_name and id exclusivity
-    meraki_content_filtering:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+    cisco.meraki.meraki_mx_content_filtering:
+      auth_key: '{{ auth_key }}'
+      org_name: '{{ test_org_name }}'
+      net_name: '{{ test_net_name }}'
       net_id: 12345
       state: present
       allowed_urls:
         - "http://www.ansible.com/*"
     register: net_exclusive
-    ignore_errors: yes    
+    failed_when: false
+    changed_when: false
 
-  - assert:
+  - name: Assert net_exclusive
+    ansible.builtin.assert:
       that:
         - 'net_exclusive.msg == "net_name and net_id are mutually exclusive"'
 
   - name: Set blocked URL categories for idempotency test
-    meraki_content_filtering:
-      state: present
-      auth_key: "{{ auth_key }}"
-      org_name: "{{ test_org_name }}"
-      net_name: "{{ test_net_name }}"
-      category_list_size: full list
-      blocked_categories:
-        - "Abused Drugs"
-        - "Adult and Pornography"
-        - "Alcohol and Tobacco"
-        - "Bot Nets"
-        - "Confirmed SPAM Sources"
-        - "Gambling"
-        - "Hacking"
-        - "Hate and Racism"
-        - "Illegal"
-        - "Malware Sites"
-        - "Open HTTP Proxies"
-        - "Peer to Peer"
-        - "Phishing and Other Frauds"
-        - "Proxy Avoidance and Anonymizers"
-    delegate_to: localhost
-
-  - name: Set blocked URL categories with idempotency
-    meraki_content_filtering:
+    cisco.meraki.meraki_mx_content_filtering:
       state: present
       auth_key: "{{ auth_key }}"
       org_name: "{{ test_org_name }}"
       net_name: "{{ test_net_name }}"
       category_list_size: full list
       blocked_categories:
-        - "Abused Drugs"
-        - "Adult and Pornography"
-        - "Alcohol and Tobacco"
-        - "Bot Nets"
-        - "Confirmed SPAM Sources"
-        - "Gambling"
-        - "Hacking"
-        - "Hate and Racism"
-        - "Illegal"
-        - "Malware Sites"
-        - "Open HTTP Proxies"
-        - "Peer to Peer"
-        - "Phishing and Other Frauds"
-        - "Proxy Avoidance and Anonymizers"
+        - "Adult"
```

Comment:
The blocked categories has been changed. 
When I curled the categories:
```bash
curl -L --request GET \
    --url https://api.meraki.com/api/v1/networks/"$MERAKI_NET_ID"/appliance/contentFiltering/categories \
    --header 'Content-Type: application/json' \
    --header 'Accept: application/json' \
    --header 'X-Cisco-Meraki-API-Key: '"$MERAKI_API_KEY"'' |
    jq .categories[].name |
    sort -u
```
I got so much different values.